Neos.Flow/Classes/Composer/InstallerScripts.php

```diff
@@ -56,17 +77,25 @@ public static function postUpdateAndInstall(Event $event)
         $packageManager->rescanPackages();
 
         chmod('flow', 0755);
+
+        self::$postUpdateAndInstallAlreadyRun = true;
     }
 
     /**
      * Calls actions and install scripts provided by installed packages.
      *
      * @param PackageEvent $event
      * @return void
+     * @throws Exception\InvalidConfigurationException
      * @throws Exception\UnexpectedOperationException
+     * @throws \Neos\Utility\Exception\FilesException
      */
-    public static function postPackageUpdateAndInstall(PackageEvent $event)
+    public static function postPackageUpdateAndInstall(PackageEvent $event): void
     {
+        if (self::$postPackageUpdateAndInstallAlreadyRun) {
```

Comment:
Just one question, which I'm not sure about: Is this script run once after each installed/updated package, or once for each package after they are all installed/updated?

Reply:
The script is triggered when all packages are installed updated for every of that packages